packages/core/src/types/redocly-yaml.ts

```diff
@@ -1,13 +1,125 @@
 import { NodeType, listOf } from '.';
 import { omitObjectProps, pickObjectProps } from '../utils';
+const customRulesRegexp = /^[a-zA-Z0-9]+\//;
```

Comment:
```suggestion
const customRulesRegexp = /^[a-zA-Z0-9\-]+\/[a-zA-Z0-9\-]+$/;
```

Reply:
I don't think it's going to be much faster (unless we expect a looot of looong rules). And it looks cumbersome to me.

Reply:
We can move it into a function. Regexp is usually much harder to read than code (easy to write though).

But well, I don't think we need full parse here, it was just example. It can be as simple as:

```ts
function isCustomRuleId(id: string) {
  return id.includes('/');
}
```

and then instead of `customRulesRegexp.test(key)` we can use `isCustomRuleId(key)`

Reply:
I like this. Later we can extend the function if we need.

Reply:
Added.